README.md

```diff
@@ -21,6 +21,7 @@ library to compile the native extensions.
 ### OSX
 
     xcode-select --install
+    rvm install 1.9
```

Comment:
well, OSX ships with ruby 2.0.0 installed. In order to get 1.9, `rvm` is the way to go. Similar to `nvm`, I would argue that `rvm` is a quintessential dependency to ruby version management.